BankChain/app/src/androidTest/java/nl/tudelft/ewi/ds/bankchain/bank/bunq/BunqTest.java

```diff
@@ -144,4 +153,100 @@ public void testExceptionConfirmationNotFuture() {
 
         assertTrue(bank.confirmException(thr) == thr);
     }
+
+    @Test
+    public void testGetUsers() throws ExecutionException, InterruptedException {
```

Comment:
We agreed to not test any bad code paths, since we are measured by line coverage instead of branch coverage.